processing/src/main/java/io/druid/segment/filter/Filters.java

```diff
@@ -157,6 +164,45 @@ public void remove()
     );
   }
 
+  public static ValueMatcher getLongValueMatcher(
+      final LongColumnSelector longSelector,
+      Comparable value
+  )
+  {
+    try {
+      if (value == null) {
+        return new BooleanValueMatcher(false);
+      }
+      final long longValue = Long.parseLong(value.toString());
```

Comment:
`Longs.tryParse` returns null instead of throwing an exception, would be a bit cleaner to use that.